Gaijinizer patch/CommonEvents_663.txt

```diff
@@ -3,386 +3,508 @@
+> CONTEXT: CommonEvents_663/557/Dialogue/enemy29
+Aaah, delicious... ♡
 
+I get to have your energy all to myself from now
+on... ♡#Ron
 > END STRING
 
 > BEGIN STRING
 \\N[1]くんも、いっぱい愉しんで…♡
 
 文字通り、時間は無限にあるんだから…
-> CONTEXT: CommonEvents_663/561/Dialogue/enemy29 < UNTRANSLATED
+> CONTEXT: CommonEvents_663/561/Dialogue/enemy29
+You enjoy yourself too, \\N[1]-kun... ♡
 
+We literally have infinite time...#Ron
 > END STRING
 
 > BEGIN STRING
 ……………やぁ
 
 探したよ、急にいなくなるんだもの
-> CONTEXT: CommonEvents_663/593/Dialogue/enemy29 < UNTRANSLATED
+> CONTEXT: CommonEvents_663/593/Dialogue/enemy29
+...Hey, I was looking for you.
 
+You suddenly went and vanished on me.#Ron
 > END STRING
 
 > BEGIN STRING
 『フェードアウト』『場面転換』……
 
 『そちら側』に安全が徹底してあるのは知ってたけど
 実際に意識して見るのは初めてだったから、驚いたなぁ
-> CONTEXT: CommonEvents_663/597/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: CommonEvents_663/597/Dialogue/enemy29
+"Fade-to-black", "scene change"...
+I knew you were completely safe on your side, but
+I was a little surprised actually consciously saw
+you for the first time.#RonIffy
 > END STRING
 
 > BEGIN STRING
 自然と『心蝕』を最小限に抑えるための機能……
 
 うまく考えられてるよね、昔から。
-> CONTEXT: CommonEvents_663/602/Dialogue/enemy29 < UNTRANSLATED
+> CONTEXT: CommonEvents_663/602/Dialogue/enemy29
+A function to naturally suppress corruption...
 
+They put a lot of thought into this, huh...?#RonIffy
 > END STRING
 
 > BEGIN STRING
 ………………え？
 \\N[1]くんはどうした？
 
 もう喰ってしまったのかって？　
-> CONTEXT: CommonEvents_663/607/Dialogue/enemy29 < UNTRANSLATED
+> CONTEXT: CommonEvents_663/607/Dialogue/enemy29
+......Hm?
+What happened to \\N[1]-kun?
 
+I already ate him.#Ron
 > END STRING
 
 > BEGIN STRING
 フフフフ………そうだねぇ……
 
 ちょっと『まばたき』してみてよ♡
-> CONTEXT: CommonEvents_663/612/Dialogue/enemy29 < UNTRANSLATED
+> CONTEXT: CommonEvents_663/612/Dialogue/enemy29
+Fufufu... Let's see...
 
+Close your eyes for just a moment. ♡#Ron
 > END STRING
 
 > BEGIN STRING
 じゃーーん♡♡
 
 \\N[1]くんは、いまだに絶賛ボクのオモチャ！
 休むことなくイき続けてるよ♡♡
-> CONTEXT: CommonEvents_663/637/Dialogue/enemy29 < UNTRANSLATED
+> CONTEXT: CommonEvents_663/637/Dialogue/enemy29
+Ta-daa... ♡♡
 
+I present to you, \\N[1]-kun!! My highly-rated
+toy that never stops cumming. ♡♡#Ron
 > END STRING
 
 > BEGIN STRING
 ただ、ボクもあちこちに移動したりしてるから
 たまにこうやって『居ない』って認識にしてるの♡
 
 なかなか便利でしょ？
-> CONTEXT: CommonEvents_663/642/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: CommonEvents_663/642/Dialogue/enemy29
+But because I carry him around wherever I go,
+I'll sometimes change awareness so he's "not
+there". ♡
+Pretty convenient, right?#RonIffyWording
 > END STRING
 
 > BEGIN STRING
 これなら、ボクが複数オモチャを持っていても
 好きなときに、好きな相手と遊べるんだ♡
 
 『居ない』から、他のオモチャが待たされる事もない
-> CONTEXT: CommonEvents_663/647/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: CommonEvents_663/647/Dialogue/enemy29
+That way, I can have multiple toys, and bring the
+one I want to play with into being whenever I
+want. Because they "don't exist", the other toys
+don't have to actually wait.#RonIffy
 > END STRING
```

Comment:
```
That way, I can have multiple toys, and bring the
one I want to play with into being whenever I
want. Because they "don't exist", the other toys
don't feel as if they've been waiting.
``